holidays/groups/islamic.py

```
@@ -328,6 +328,16 @@ def _add_laylat_al_qadr_day(self, name):
             name, self._islamic_calendar.laylat_al_qadr_dates(self._year)
         )
 
+    def _add_jumuatul_wida(self, name):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_add_jumuatul_wida`

(ANN202)